### ISSUE TITLE
skipif mark from parametrize hides module level skipif mark

### ISSUE THREAD

**Issue by adler-j** [pytest-dev/pytest#1540]
Using pytestmark in combination with fixtures that has a skip causes the pytestmark to be ignored.

MWE:

``` python
import pytest

pytestmark = pytest.mark.skipif(True, reason='skip all')

# skipifs inside fixture params
params = [pytest.mark.skipif(False, reason='dont skip')('parameter')]


@pytest.fixture(params=params)
def parameter(request):
    return request.param


def test_1(parameter):
    assert True
```

returns:

``` bash
user$ py.test -v
======================================== test session starts =========================================
platform linux2 -- Python 2.7.6, pytest-2.8.3, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: xxx, inifile: 
plugins: cov-2.2.0, pep8-1.0.6
collected 1 items 

example_test.py::test_1[parameter] PASSED

====================================== 1 passed in 0.00 seconds ======================================
```

**Comment by RonnyPfannschmidt** [comment]
i'm not sure if we already added support for obtaining mark from parameters that come via fixtures

can you verify against the 2.9 series?

**Comment by adler-j** [comment]
Still happens with pytest 2.9:

```bash
======================================== test session starts =========================================
platform linux2 -- Python 2.7.6, pytest-2.9.1, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: xxx, inifile: 
plugins: cov-2.2.0, pep8-1.0.6
collected 1 items 

example_test.py::test_1[parameter] PASSED

====================================== 1 passed in 0.00 seconds ======================================
```

**Comment by RonnyPfannschmidt** [comment]
after analysis this one has to be solved after landing #1534

**Comment by RonnyPfannschmidt** [comment]
this requires #2522

**Comment by RonnyPfannschmidt** [comment]
#2672 will be the first step to fixing this

currently marks from parametrize just override other marks, and thats wrong

**Comment by RonnyPfannschmidt** [comment]
fixed in #3317

**Comment by adler-j** [comment]
Thank you so much for fixing this!
